### PR TITLE
Enable Background Playback

### PR DESCRIPTION
**Pull request by brunopinheiro**
Changes to enable the external playback keep running on background

### CODE REVIEW

**Comment by gabiduarte** [comment]
[Habilitar Playback Externo em Background](https://trello.com/c/LVRNIZ5n/34-habilitar-playback-externo-em-background)

File: Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift [hunk 1]
```diff
@@ -5,350 +5,359 @@ enum PlaybackState {
 }
 
 open class AVFoundationPlayback: Playback {
```
Comment (diegomarcon) [inline Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift]: Shouldn't we be using 4 spaces for tabs?

Comment (brunopinheiro) [inline Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift]: Just pushed a new commit fixing the tabs 😄

Comment (diegomarcon) [inline Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift]: 👍

**Pull request merged**